testng-core/src/main/java/org/testng/internal/WrappedTestNGMethod.java

```diff
@@ -364,6 +365,11 @@ public String getQualifiedName() {
     return testNGMethod.getQualifiedName();
   }
 
+  @Override
+  public UUID getInstanceId() {
+    return testNGMethod.getInstanceId();
+  }
```

Comment:
I believe this is taken care of because `ITestNGMethod` has only 2 implementations in TestNG viz.,, 

* `BaseTestMethod` - this already has an implementation 
* `WrappedTestNGMethod` - This just delegates the call to the embedded `ITestNGMethod`